ols/src/query_helpers/question_validator.py

```diff
@@ -16,6 +15,14 @@
 class QuestionValidator(QueryHelper):
     """This class is responsible for validating questions and providing one-word responses."""
 
+    @property
+    def llm_params(self) -> dict:
+        """Return the parameters for the LLM."""
+        return {
+            "min_new_tokens": 1,
+            "max_new_tokens": 4,
+        }
```

Comment:
i don't understand the choice to define a function to return what is effectively a constant.  Seems unnecessarily complicated.  The caller of llm_loader can simply pass the llm parameters they want which makes it much more obvious at call time what args are being passed.

Reply:
yes i think that would be better

Reply:
changed